rialto_airflow/dags/harvest.py

```diff
@@ -128,10 +132,11 @@ def pubs_to_contribs(pubs, doi_sunet_pickle, snapshot):
         """
         Get contributions from publications.
         """
-        output = snapshot.path / "contributions.parquet"
-        create_contribs(pubs, doi_sunet_pickle, output)
+        authors_csv = snapshot.path / "authors.csv"
+        contribs_path = snapshot.path / "contributions.parquet"
```

Comment:
this seemed like a clearer name, and lined up better with the param name of the `create_contribs` function